OneSignalSDK/unittest/src/test/java/com/test/onesignal/GenerateNotificationRunner.java

```diff
@@ -144,13 +143,12 @@
         },
         sdk = 21
 )
-
 @RunWith(RobolectricTestRunner.class)
 public class GenerateNotificationRunner {
-   
+
```

Comment:
We might want to omit the formatting space fixes in this PR since this is going to into master. This way we can avoid extra conflicts this could created with the 4.0.0 branch.

Reply:
Oh you are right, good catch!